lib/utils/dataflow/src/index.js

```diff
@@ -715,11 +735,11 @@ const mapper = (mapfn, opt) => {
       return {
         body: dbclient.undbify(doc)
       };
-    }));
+    });
 
   // Retrieve error docs
   if(opt.error && opt.error.get)
-    routes.get(opt.error.get, throttle(function *(req) {
```

Comment:
Wouldn't it be better if we change throttle to fail the requests then? 

I imagine it can:
* store request and timestamp
* reject/ignore requests older than some time
* can be configured with some timeout